api/types/client.go

```diff
@@ -235,10 +235,21 @@ type ImageImportOptions struct {
 	Platform string   // Platform is the target platform of the image
 }
 
-// ImageListOptions holds parameters to filter the list of images with.
+// ImageListOptions holds parameters to list images with.
 type ImageListOptions struct {
-	All     bool
+	// All controls whether all images in the graph are filtered, or just
+	// the heads.
+	All bool
+
+	// Filters is a JSON-encoded set of filter arguments.
+	// filter is a shell glob string applied to repository names.
```

Comment:
`filter is a shell glob string applied to repository names.` looks to be a description for the legacy `?filter` parameter. Probably should be removed here, as this describes the "filters" as a whole (not that parameter)

We can move that comment to api/server/router/image/image_routes.go to describe the property there, in case it's useful.